week1_intro/primer/recap_pytorch.ipynb

```diff
@@ -1078,17 +1088,17 @@
     "  - Softmax (exp over sum of exps) can be implemented manually or as `nn.Softmax` (layer) or `F.softmax` (function)\n",
     "  - Probably better to use STOCHASTIC gradient descent (minibatch) for greater speed\n",
     "    - You can also try momentum/rmsprop/adawhatever\n",
-    "    - in which case the dataset should probably be shuffled (or use random subsamples on each iteration)\n",
+    "    - in this case the dataset should probably be shuffled (or use random subsamples on each iteration)\n",
     "* Add a hidden layer. Now your logistic regression uses hidden neurons instead of inputs.\n",
     "  - Hidden layer uses the same math as output layer (ex-logistic regression), but uses some nonlinearity (e.g. sigmoid) instead of softmax\n",
     "  - You need to train both layers, not just the output layer :)\n",
     "  - 50 hidden neurons and a sigmoid nonlinearity will do for a start. Many ways to improve. \n",
     "  - In ideal case this totals to 2 `torch.matmul`'s, 1 softmax and 1 ReLU/sigmoid\n",
     "  - __Make sure this neural network works better than logistic regression!__\n",
     "  \n",
-    "* Now's the time to try improving the network. Consider layers (size, neuron count), nonlinearities, optimization methods, initialization — whatever you want, but please avoid convolutions for now.\n",
+    "* Now's the time to try improving the network. Consider layers (size, neuron count), nonlinearities, optimization methods, initialization — whatever you want, but, please, avoid convolutions for now.\n",
     "  \n",
-    "* If anything seems wrong, try going through one step of training and printing everything you compute.\n",
+    "* If something seems wrong, try going through each step of training and print everything you compute.\n",
```

Comment:
I think the original text meant "try going through a single training step and print everything you compute".